src/main/java/se/sundsvall/esigning/api/model/Signatory.java

```diff
@@ -0,0 +1,164 @@
+package se.sundsvall.esigning.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import se.sundsvall.dept44.common.validators.annotation.OneOf;
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+
+@Schema(description = "A party assigned as signatory to the signing.")
+public class Signatory {
+
+	@Schema(description = "The name of the party.", example = "John Doe")
+	private String name;
+
+	@Schema(description = "The uuid of the party.", example = "550e8400-e29b-41d4-a716-446655440000", requiredMode = REQUIRED)
+	@ValidUuid
+	private String partyId;
+
+	@Schema(description = "Optional message for the signature request emails for the specific party. Overwrites the default message in the signing request when provided.")
+	@Valid
+	private NotificationMessage notificationMessage;
+
+	@Schema(description = "The title for the party.", example = "CEO")
+	private String title;
+
+	@Schema(description = "The email for the party.", example = "john.doe@sundsvall.se", requiredMode = REQUIRED)
+	@Email
+	@NotNull
```

Comment:
Should probably be ```@NotBlank```?

Reply:
This is probably covered by the ```@Email```

Reply:
`@NotNull` is needed as null values is not validated by `@Email` annotation (hence null passes as valid email). If a value is present (for example blank string) then it is validated by `@Email` annotation (and blank string is not a valid email).